base/distributed/cluster.jl

```diff
@@ -167,8 +175,6 @@ function start_worker(out::IO, cookie::AbstractString)
     print(out, LPROC.bind_addr)
     print(out, '\n')
     flush(out)
-    # close STDIN; workers will not use it
```

Comment:
it was commented out in https://github.com/JuliaLang/julia/commit/b5379c3a8c5f6623a6e463ed765d4740a9e145dc with zero explanation of why